src/app/app.component.ts

```diff
@@ -11,177 +11,257 @@ import { ThemesService } from './core/themes/themes.service';
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss'],
 })
 export class AppComponent implements OnInit {
-
-  constructor(public settings: SettingsService,
-              private matIconRegistry: MatIconRegistry,
-              private domSanitizer: DomSanitizer,
-              private urlHistoryService: UrlService,
-              private themeService: ThemesService,
-              private store: Store) {
+  constructor(
+    public settings: SettingsService,
```

Comment:
I prefer previous alignment. And i would like to stick with old one at least in this repo, because I'm the only person that works on switchboard atm.